packages/xo-web/src/xo-app/self/index.js

```diff
@@ -597,9 +597,12 @@ class ResourceSet extends Component {
   _renderDisplay = () => {
     const { resourceSet } = this.props
     const resolvedIpPools = mapKeys(this.props.ipPools, 'id')
-    const { limits, ipPools, subjects, objectsByType } = resourceSet
+    const { limits, id, ipPools, subjects, objectsByType } = resourceSet
 
     return [
+      <li key={id} className='list-group-item'>
+        <span className='tag tag-default'>id</span> {id}
```

Comment:
It's better to display `id` next to the resource name.

Reply:
@Rajaa-BARHTAOUI Do you want it where I drawed the arrow on the picture ? 
![self id](https://user-images.githubusercontent.com/25125613/56362462-3622c280-61ea-11e9-909e-c66f8900a9f3.png)

Reply:
yes, I think it's better.

Reply:
According to UI/UX rules, the user must see the minimum information needed. Displaying an ID beside the resource set name would be a little bit confusing. In addition the area where you want to display the ID is styled as `cursor: pointer` which means that we can't copy the ID from there :(

Reply:
I think displaying the identifier behind the name in between braces is similar to the rest of the application.

Maybe it should use `Copiable` indeed.

@pdonias What's your take on this?

Reply:
I don't think we even need to make it copiable. It's just meant to differentiate the resource sets from one another if their name are too similar or to make it easier to investigate issues, kind of like we do with backup jobs. So yes, putting the ID (or even a substring of it) in braces behind the name should be fine.

Reply:
Also, we need to see it without having to uncollapse the resource set.

Reply:
@pdonias So I understand from what you said that you investigate issues without copying ressource set id ?

Reply:
Yes. Let's say a user triggered an error creating VMs with Self Service. The log will only report the ID of the resource set and you only need the first few characters to find it in the list. Or if a user wants to swap 2 resource sets' names, the list will dynamically reorder alphabetically so it can help the user keep track of the 2 resource sets while they have the same name.

It's just a tiny helper we're used to add (cf jobs, backups, logs, licenses, etc.)